docker/docker-compose.yml

```diff
@@ -168,6 +196,20 @@ services:
       - 9001:9001 # enable for UI access
     command: server --certs-dir /certs /data
 
+  superset_init:
+    environment:
+      PGPASSWORD: postgres
+      PGUSER: postgres
+      PGHOST: db
+    image: postgis/postgis:12-3.1-alpine
```

Comment:
Maybe add a comment for those that need to override this. Or, I can take a pass at a universal postgis Dockerfile (which worked for Kevin and me). Don't want to derail the main goal here, though.